src/topology/instances/real.lean

```diff
@@ -285,11 +284,6 @@ eq_Icc_of_connected_compact ⟨(nonempty_Icc.2 hab).image f, is_preconnected_Icc
 
 end
 
-instance reals_semimodule : has_continuous_smul ℝ ℝ := ⟨continuous_mul⟩
```

Comment:
To confirm, this instance already exists as `has_contnuous_mul.has_continuous_smul` [sic, #6921]